src/ActionRow/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
Do we need to export all React library for this d.ts file? I propose to export only React module.